lib/player_factory.rb

```diff
@@ -9,8 +10,12 @@ def initialize(display)
   
   def get_players_for_game_type(game_type)
     case game_type
-    when GameType::HVH
+    when GameTypeOptions::HVH
       players = Hash[Mark::X, HumanPlayer.new(Mark::X, display), Mark::O, HumanPlayer.new(Mark::O, display)]
+    when GameTypeOptions::HVB
+      players = Hash[Mark::X, HumanPlayer.new(Mark::X, display), Mark::O, BeatableAIPlayer.new(Mark::O, display)]
+    when GameTypeOptions::BVH
+      players = Hash[Mark::X, BeatableAIPlayer.new(Mark::X, display), Mark::O, HumanPlayer.new(Mark::O, display)]
```

Comment:
no default? GameTypeOptions could add another constant and this would just return nil and we'd get a runtime error far away from here